setup.cfg

```diff
@@ -23,9 +23,11 @@ exclude =
     .scratch,
     _vendor,
     data
-select = E,W,F
-ignore = W504
-
+per-file-ignores =
+    # B011: Do not call assert False since python -O removes these calls
+    tests/*: B011
+    # TODO: Remove this when fixing flake8-bugbear warnings in source
+    src/pip/*: B, B9
```

Comment:
should be able to just use `B` here since it's prefix-matched

Reply:
Good idea. Removed the `B9`